app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -305,15 +305,14 @@ def unconfirmed_phone?
   end
 
   def phone_view_data
```

Comment:
I was like "oh nothing seems to call this" and then I remembered what mixin we're in with dynamic sends everywhere 😎 😭 

https://github.com/18F/identity-idp/blob/38b37f8e13bf35186f2e42608a1a582990e9a4e7/app/controllers/concerns/two_factor_authenticatable_methods.rb#L379

Reply:
yep, would like to redo this whole thing

Reply:
BIG SAME